edx_sga/sga.py

```diff
@@ -563,6 +563,14 @@ def enter_grade(self, request, suffix=''):
         module_id = request.params.get('module_id')
         student_id = request.params.get('student_id')
 
+        if not score:
+            return Response(
+                json_body=self.validate_score_message(
+                    module.course_id,
+                    module.student.username
```

Comment:
I think module is not yet defined in this context. I think line 575 introduces it.